pkg/envoy/resources.go

```diff
@@ -149,40 +159,32 @@ func (cache *NPHDSCache) handleIPUpsert(npHost *envoyAPI.NetworkPolicyHosts, ide
 		HostAddresses: hostAddresses,
 	}
 	if err := newNpHost.Validate(); err != nil {
-		scopedLog.WithError(err).WithFields(logrus.Fields{
-			logfields.XDSResource: newNpHost.String(),
-		}).Warning("Could not validate NPHDS resource update on upsert")
-		return
+		return errors.New(fmt.Sprintf("Could not validate NPHDS resource update on upsert: %s (%s)", err.Error(), newNpHost.String()))
```

Comment:
Could probably use `fmt.Errorf()` and use the `%w` specifier to wrap the underlying error (error must be the last argument).

Reply:
I had forgot about that, thanks!